lib/EnsEMBL/REST/Model/LDFeatureContainer.pm

```diff
@@ -81,6 +81,10 @@ sub fetch_LDFeatureContainer_variation_name {
 sub fetch_LDFeatureContainer_slice {
   my ($self, $slice, $population_name) = @_;
   Catalyst::Exception->throw("No region given. Please specify a region to retrieve from this service.") if ! $slice;
+  if (slice_overlaps_mhc_region($slice)) {
+    my $mhc_region = get_mhc_region($slice);
+    Catalyst::Exception->throw("Specified region overlaps MHC region $mhc_region and can therefore not be greater than 10KB.") if ($slice->length > 10_000);
```

Comment:
Please change this if statement to block syntax. I know that postfix syntax is generally tolerated for terminal statements like this one but this line is so long that having to look all the way to the right to see that it is conditional really breaks the flow of reading the code.
Consider changing this in lines: 83 and88 as well while at it, seeing as they are so close to a new code block.